frontend/elements/src/core/menu/menu-tab.ts

```diff
@@ -38,12 +40,21 @@ export class _ extends LitElement {
     `];
   }
 
+  @property()
+  path:string = ""; 
```

Comment:
`path` is a confusing name for menu property - should be `uiIconPath` or something like that.